### DISCUSSION TITLE
Bug with reader both old and new

### DISCUSSION THREAD

**Discussion by Dmnboruto** [jakepurple13/OtakuWorld#350] [Ideas]
The reader both old and new bugs when you tap the next chapter button the manga doesn't load until you reload it 😑 and please just saying but I think you should make a discord server so that the people can share bugs and just talk you know

**Comment by jakepurple13** [comment]
What phone and version of android are you using and on?
Also, what version of the app are you using?

Does this also happen for every manga or just certain ones? Also, what source are you using?

I've been thinking about making a discord server. I'll look into it today maybe.

**Reply by Dmnboruto** [reply]
I'm using Samsung tab a it only bugs on one punch man for me but it could just be my device.

**Reply by Dmnboruto** [reply]
I think it just my device cause mine ain't the best

**Reply by jakepurple13** [reply]
What's the source you're using?

**Reply by Dmnboruto** [reply]
I am using MANGA4LIFE
